### PR TITLE
[Install] Guideline and PHP version update

### LINKED ISSUE #6703: [CentOS Install] Readme and install script minor modifications
Overall, the CentOs install works relatively well, except for some problems with permissions:
* sudo is required in the [install.sh tool L118](https://github.com/aces/Loris/blob/23.0-release/tools/install.sh#L118)

Some typo were also found in the install README (loris/docs/config/apache-site instead of loris/docs/config/apache2-site) and additional steps need to be added.
 
Related to  #4824

### LINKED ISSUE #6722: [docs] Update Readme specs and re-test Install on PHP 7.4
This should be using PHP 7.4, not 7.3. 7.3 is end of life in 5 months.

_Originally posted by @driusan in https://github.com/aces/Loris/pull/6704_

This update should apply across the board to the 23 release - docs and probably also install testing.

### PR DESCRIPTION
**Pull request by laemtl**
Overall, the CentOS install works relatively well, except for some problems with permissions:
* sudo is required in the install.sh tool L118

Some typos were also found in the install README (loris/docs/config/apache-site instead of loris/docs/config/apache2-site), and steps added. 

Also updated the references of php7.3 to php7.4 across the documentation (Ubuntu/CentOS).

* Resolves #6703
* Resolves #6722
* Related to #4824

### CODE REVIEW

**Comment by christinerogers** [comment]
requesting review from @johnsaigle given this is adding another `sudo` use in the install script.

**Comment by johnsaigle** [comment]
I'm not very familiar with CentOS but based on what [Melanie wrote](https://github.com/aces/Loris/issues/4824#issuecomment-559511372) about this problem earlier, it sounds like additional set up is required in advance. Specifically, it sounds like the installing user needs some sudo permissions which can then be removed post-install.

Since these instructions (in contrast to the Ubuntu instructions) seem to already require `sudo` I think it's OK to have another one here.

Further work should probably be done to streamline the sudo process for CentOS.

**Comment by driusan** [comment]
@PapillonMcGill correct me if I'm wrong but last release didn't you point out that sudo wasn't available by default in CentOS?

**Comment by PapillonMcGill** [comment]
@driusan There is a small note on install page for CentOS about that.

File: docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md [hunk 1]
```diff
@@ -37,7 +37,11 @@ sudo yum install epel-release
 sudo yum install http://rpms.remirepo.net/enterprise/remi-release-7.rpm
 sudo yum install yum-utils
 sudo yum update
-sudo yum --enablerepo=remi-php73 install php php-pdo php-pdo_mysql php73-php-fpm php73-php-gd php73-php-json php73-php-mbstring php73-php-mysqlnd php73-php-xml php73-php-xmlrpc php73-php-opcache php73-php-mysql
+
+# By default, the repository for PHP 5.4 is enabled
+sudo yum-config-manager --disable remi-php54
+sudo yum-config-manager --enable remi-php73
```
Comment (driusan) [inline docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md:43]: This should be using PHP 7.4, not 7.3. 7.3 is end of life in 5 months.

Comment (christinerogers) [reply docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md:43]: ok -- this update should then also be made in the main Readme and the docs, and possibly re-test the installs.

Comment (laemtl) [reply docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md]: Readme updated. This need to be tested.

Comment (christinerogers) [reply docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md]: great, thanks @laemtl - pls test when you have a moment.

Comment (laemtl) [reply docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md]: Replace 7.3 in the Ubuntu install file too.

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Deploy and log in with username *admin* and the password that's set up during de
 
  * Apache **2.4** or higher
  * MySQL >= 5.7 (or MariaDB >= 10.3) 
- * PHP <b>7.3</b> or higher
+ * PHP <b>7.4</b> or higher
```
Comment (driusan) [inline README.md:33]: LORIS still works with 7.3, 7.4 isn't a requirement. This shouldn't be changed (Travis runs both 7.3 and 7.4), but install instructions shouldn't be telling people to install a version that will soon be out of date.

(Same with other sections of this PR that are now modified.)

**Comment by christinerogers** [comment]
@driusan  final re/review pls - changes should have been addressed.  

Thank you all!

**Comment by laemtl** [comment]
Installation tested with php7.4 on CentOS and works without any issue.

**Pull request merged**